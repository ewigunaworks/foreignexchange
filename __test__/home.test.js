// Copyright 2004-present Facebook. All Rights Reserved.

'use strict';

import React from 'react';
import Home from '../src/pages/home';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});