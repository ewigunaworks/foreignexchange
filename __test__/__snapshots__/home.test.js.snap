// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <section
    id="intro"
  >
    <div
      class="intro-text"
    >
      <h2>
        Welcome to
      </h2>
      <p>
        Foreign Exchange Currency Apps
      </p>
      <a
        class="btn-get-started scrollto"
        href="#mainmenu"
      >
        Get Started
      </a>
    </div>
  </section>,
  <main
    id="main"
  >
    <section
      class="section-bg"
      id="mainmenu"
    >
      <div
        class="container-fluid py-4"
      >
        <div
          class="section-header"
        >
          <h3
            class="section-title"
          >
            Main Menu
          </h3>
          <span
            class="section-divider"
          />
          <p
            class="section-description"
          >
            Foreign Exchange Currency App Exercise
          </p>
        </div>
        <div
          class="row"
        >
          <div
            class="col-12 text-center"
          >
            <p
              class="mb-1"
            >
              <em>
                USD - 
                US Dollar
              </em>
            </p>
            <div
              class="row align-items-center justify-content-center m-2"
            >
              <p
                class="mb-1"
              >
                <b>
                  USD   
                </b>
              </p>
              <input
                class="text-right"
                id="currency"
                name="currency"
                onChange={[Function]}
                placeholder="Enter Nominal"
                type="text"
                value="10"
              />
            </div>
          </div>
          <hr />
        </div>
        <div
          class="row"
        >
          <div
            class="col-lg-4 col-md-6 col-sm-12 mb-1"
          >
            <div
              class="card"
            >
              <div
                class="card-body"
              >
                <div
                  class="row"
                >
                  <div
                    class="col-10"
                  >
                    <div
                      class="row"
                    >
                      <div
                        class="col-6"
                      >
                        <p
                          class="mb-0"
                        >
                          <b>
                            IDR
                          </b>
                        </p>
                      </div>
                      <div
                        class="col-6"
                      >
                        <p
                          class="font-25 mb-0"
                        >
                          NaN
                        </p>
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-12"
                      >
                        <p
                          class="mb-0 font-12"
                        >
                          <b>
                            <em>
                              IDR
                               - 
                              Rupiah
                            </em>
                          </b>
                        </p>
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-12"
                      >
                        1 USD = 
                        IDR
                         
                        NaN
                         
                      </div>
                    </div>
                  </div>
                  <div
                    class="col-2"
                  >
                    <a
                      class="btn btn-danger"
                      href="javascript:void(0)"
                      onClick={[Function]}
                    >
                      -
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="col-lg-4 col-md-6 col-sm-12 mb-1"
          >
            <div
              class="card"
            >
              <div
                class="card-body"
              >
                <div
                  class="row"
                >
                  <div
                    class="col-10"
                  >
                    <div
                      class="row"
                    >
                      <div
                        class="col-6"
                      >
                        <p
                          class="mb-0"
                        >
                          <b>
                            EUR
                          </b>
                        </p>
                      </div>
                      <div
                        class="col-6"
                      >
                        <p
                          class="font-25 mb-0"
                        >
                          NaN
                        </p>
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-12"
                      >
                        <p
                          class="mb-0 font-12"
                        >
                          <b>
                            <em>
                              EUR
                               - 
                              Euro
                            </em>
                          </b>
                        </p>
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-12"
                      >
                        1 USD = 
                        EUR
                         
                        NaN
                         
                      </div>
                    </div>
                  </div>
                  <div
                    class="col-2"
                  >
                    <a
                      class="btn btn-danger"
                      href="javascript:void(0)"
                      onClick={[Function]}
                    >
                      -
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="col-lg-4 col-md-6 col-sm-12 mb-1"
          >
            <div
              class="card"
            >
              <div
                class="card-body"
              >
                <div
                  class="row"
                >
                  <div
                    class="col-10"
                  >
                    <div
                      class="row"
                    >
                      <div
                        class="col-6"
                      >
                        <p
                          class="mb-0"
                        >
                          <b>
                            GBP
                          </b>
                        </p>
                      </div>
                      <div
                        class="col-6"
                      >
                        <p
                          class="font-25 mb-0"
                        >
                          NaN
                        </p>
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-12"
                      >
                        <p
                          class="mb-0 font-12"
                        >
                          <b>
                            <em>
                              GBP
                               - 
                              Pound Sterling
                            </em>
                          </b>
                        </p>
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-12"
                      >
                        1 USD = 
                        GBP
                         
                        NaN
                         
                      </div>
                    </div>
                  </div>
                  <div
                    class="col-2"
                  >
                    <a
                      class="btn btn-danger"
                      href="javascript:void(0)"
                      onClick={[Function]}
                    >
                      -
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="col-lg-4 col-md-6 col-sm-12 mb-1"
          >
            <div
              class="card"
            >
              <div
                class="card-body"
              >
                <div
                  class="row"
                >
                  <div
                    class="col-10"
                  >
                    <div
                      class="row"
                    >
                      <div
                        class="col-6"
                      >
                        <p
                          class="mb-0"
                        >
                          <b>
                            SGD
                          </b>
                        </p>
                      </div>
                      <div
                        class="col-6"
                      >
                        <p
                          class="font-25 mb-0"
                        >
                          NaN
                        </p>
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-12"
                      >
                        <p
                          class="mb-0 font-12"
                        >
                          <b>
                            <em>
                              SGD
                               - 
                              Singapore Dollar
                            </em>
                          </b>
                        </p>
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-12"
                      >
                        1 USD = 
                        SGD
                         
                        NaN
                         
                      </div>
                    </div>
                  </div>
                  <div
                    class="col-2"
                  >
                    <a
                      class="btn btn-danger"
                      href="javascript:void(0)"
                      onClick={[Function]}
                    >
                      -
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="row align-items-center justify-content-center btn-add-more pt-2"
        >
          <div
            class="col-sm-12 text-center"
          >
            <a
              class="btn btn-primary"
              href="javascript:void(0)"
            >
              Add More Currencies
            </a>
          </div>
        </div>
        <div
          class="row align-items-center justify-content-center select-currencies closed pt-2"
        >
          <div
            class="col-sm-12"
          >
            <div
              class="row align-items-center justify-content-center"
            >
              <div
                class="mx-2 w-50"
              >
                <div
                  className=" css-2b097c-container"
                  onKeyDown={[Function]}
                >
                  <div
                    className=" css-yk16xz-control"
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                  >
                    <div
                      className=" css-1hwfws3"
                    >
                      <div
                        className=" css-1wa3eu0-placeholder"
                      >
                        Select...
                      </div>
                      <div
                        className="css-w8afj7-Input"
                      >
                        <div
                          className=""
                          style={
                            Object {
                              "display": "inline-block",
                            }
                          }
                        >
                          <input
                            aria-autocomplete="list"
                            autoCapitalize="none"
                            autoComplete="off"
                            autoCorrect="off"
                            disabled={false}
                            id="react-select-2-input"
                            onBlur={[Function]}
                            onChange={[Function]}
                            onFocus={[Function]}
                            spellCheck="false"
                            style={
                              Object {
                                "background": 0,
                                "border": 0,
                                "boxSizing": "content-box",
                                "color": "inherit",
                                "fontSize": "inherit",
                                "label": "input",
                                "opacity": 1,
                                "outline": 0,
                                "padding": 0,
                                "width": "1px",
                              }
                            }
                            tabIndex="0"
                            type="text"
                            value=""
                          />
                          <div
                            style={
                              Object {
                                "height": 0,
                                "left": 0,
                                "overflow": "scroll",
                                "position": "absolute",
                                "top": 0,
                                "visibility": "hidden",
                                "whiteSpace": "pre",
                              }
                            }
                          >
                            
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      className=" css-1g48xl4-IndicatorsContainer"
                    >
                      <span
                        className=" css-1okebmr-indicatorSeparator"
                      />
                      <div
                        aria-hidden="true"
                        className=" css-tlfecz-indicatorContainer"
                        onMouseDown={[Function]}
                        onTouchEnd={[Function]}
                      >
                        <svg
                          aria-hidden="true"
                          className="css-tj5bde-Svg"
                          focusable="false"
                          height={20}
                          viewBox="0 0 20 20"
                          width={20}
                        >
                          <path
                            d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                          />
                        </svg>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <a
                class="btn btn-success mx-2"
                href="javascript:void(0)"
                onClick={[Function]}
              >
                Submit
              </a>
            </div>
          </div>
        </div>
      </div>
    </section>
  </main>,
  <footer
    id="footer"
  >
    <div
      class="container"
    >
      <div
        class="row justify-content-center align-items-center"
      >
        <div
          class="col-lg-6 text-lg-left"
        >
          <div
            class="copyright text-center"
          >
            © 2019 
            <strong>
              Erwin Wiguna
            </strong>
            .
          </div>
          <div
            class="credits"
          >
            <p
              class="text-center"
            >
              <small>
                ewings
              </small>
            </p>
          </div>
        </div>
      </div>
    </div>
  </footer>,
  <a
    class="back-to-top"
    href="#"
  >
    <i
      class="fa fa-chevron-up"
    />
  </a>,
]
`;
